app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -0,0 +1,166 @@
+class MS.Views.LoanChartsView extends Backbone.View
+
+  el: '.summary-chart'
+
+  initialize: (params) ->
+    @breakevenData = params.breakeven_data
+    @breakevenFixedCosts = @breakevenData["fixed_costs"]
+    @breakevenProductionCosts = @breakevenData["cogs"]
+    @breakevenRevenue = @breakevenData["revenue"]
+    @defaultChartOptions = {
+      width: "100%"
+    }
+
+    @loadCharts()
+
+  breakevenFixedCostsChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.fixed_costs', count: 1),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.amount'),"pattern":"","type":"number"}
+    ]
+
+    rows = []
+    for key,cost of @breakevenFixedCosts
+      name = cost.name
+      total = cost.amount
+      rows.push({"c":[{"v": name, "f":null},{"v": total, "f":null}]})
+
+    options = @defaultChartOptions
+    chartData = {"cols": columns, "rows": rows}
+    chartData = new google.visualization.DataTable(chartData);
+    chart = new google.visualization.PieChart(document.getElementById('breakeven-fixed-cost-chart'));
+    chart.draw(chartData, options);
+
+  breakevenProductProfitChart: () ->
+    chartData = {}
+    columns = [
+      {"id":"","label":I18n.t('loan.breakeven.product'),"pattern":"","type":"string"},
+      {"id":"","label":I18n.t('loan.breakeven.profit'),"pattern":"","type":"number"}
+    ]
```

Comment:
Looking at the [google charts documentation](https://google-developers.appspot.com/chart/interactive/docs/gallery/piechart), it looks like the data format can be simplified significantly. Is there a reason you chose to specify all these keys?